content/blog/anarchos/2024-04-09_an_odissey_to_port_compcert.md

```diff
@@ -0,0 +1,94 @@
++++
+type = "blog"
+title = "An odissey to port compcert"
+author = "Anarchos"
+date = "2024-04-09 09:44:28+00:00"
+tags = ["haiku", "software"]
++++
+
+
+### The goal 
+My goal is to be able to use [CompCert](http://compcert.org), a certified compiler.
+It is a compiler whose all passes are formally verified to not introduce change in the semantics from the `C` code to its translation in `asm`.
```

Comment:
"compiler whose all passes" is clumsy. Maybe something like "compiler whose passes"?

Reply:
or, alternatively "compiler whose passes all"

Reply:
Or:
```suggestion
All passes of a 'certified compiler' are formally verified to not introduce change in the semantics from the `C` code to its translation in `asm`.
```